server/src/actors/events.rs

```diff
@@ -120,102 +120,118 @@ impl EventProcessor {
 
     fn process(&self, message: &ProcessEvent) -> Result<ProcessEventResponse, ProcessingError> {
         log::trace!("processing event {}", message.event_id);
-        let mut event = Annotated::<Event>::from_json_bytes(&message.data).map_err(|error| {
-            if message.log_failed_payloads {
-                let mut event = event_from_fail(&error);
-                message.add_to_sentry_event(&mut event);
-                sentry::capture_event(event);
-            }
-
-            ProcessingError::InvalidJson(error)
-        })?;
-
-        if let Some(event) = event.value_mut() {
-            event.id = Annotated::new(message.event_id);
-        }
+        metric! {timer("event_processing.total_time"), {
```

Comment:
Do we really need this metric here if we also capture `event.processing_time` by timing `.process()` in https://github.com/getsentry/semaphore/pull/261/files#diff-46d082798bb67039abd23aa05c8c2a2bR308?

Reply:
you are right, I'will remove `event_processing.total_time`